cypress/features/regression/experimental/fieldsetClassic.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Experimental Fieldset component
```

Comment:
Add that it's related with 'classic' component

Reply:
fixed